src/main/java/com/example/demo/user/User.java

```diff
@@ -0,0 +1,94 @@
+
+package com.example.demo.user;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Collection;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "users")
+@NoArgsConstructor
+public class User  implements UserDetails{
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private Integer id;
+
+    @Column(name = "email", length = 50, nullable = false)
+    private String email;
+    @Column(name = "password", length = 72, nullable = false)
```

Comment:
It's better to store password hash instead of the password in a plain text. See for reference: https://www.baeldung.com/spring-security-registration-password-encoding-bcrypt